high_level/profile/profile_creation.md

```diff
@@ -0,0 +1,112 @@
+# Profile Creation - The technical part
+
+<!-- toc -->
+
+## What is a profile?
+
+All of the changes a user makes in Firefox, like the home page, what toolbars you use, installed addons, saved passwords and your bookmarks, are all stored in a special folder, called a profile.
+Telemetry stores archived and pending pings in the profile directory as well as metadata like the client ID.
+
+Every run of Firefox needs a profile. However a single installation can use multiple profiles for different runs.
+The profile folder is stored in a separate place from the Firefox program so that, if something ever goes wrong with Firefox, the profile information will still be there.
+
+Firefox also comes with a profile manager, a different run mode to create, migrate and delete the profiles.
+
+## Profile Behaviors
+
+In order to understand the behavior of users and base analysis on things like the profile creation date,
+it is essential to understand how a profile is created and identified by the browser.
+
+A profile can also be reset or deleted by the user.
+The user can opt-out of or opt-in to sending Telemetry data.
+
+These different cases are described in more detail in the following sections.
+
+### Profile Creation
+
+#### First use
+
+When Firefox is opened for the first time after a fresh install, without any prior Firefox profile on disk, it will create a new profile.
+Firefox chooses a random name, create the profile directory and marks the new profile as default for subsequent starts of Firefox.
+
+#### Default or Profile Manager creation
+
+On subsequent starts, Firefox tries to find the profile directory and load the profile data (bookmarks, addons, etc.).
+If not otherwise specified it uses the default (or last used) profile, finds the profile directory and starts loading files from there.
+
+The user can create a new profile through the profile manager.
+This can either be done on `about:profiles` or by starting Firefox with the `--ProfileManager` flag.
+The profile manager will ask for a name for the profile and picks a new directory for it.
+The directory can be changed by the user to another new or even an existing directory.
+If that directory does not exist, it is created.
+Additional files for bookmarks are placed inside this directory.
+
+#### Command-line start
+
+Firefox can be started on the command line with a path to a profile directory: `firefox --profile path/to/directory`.
+In that case the passed profile directory is, if not already existing, created. No files inside that directory are written at launch.
+
```

Comment:
sorry, confused by the statement "No files inside that directory are written at launch."

Reply:
Also, one important distinction that we might want to mention is how with this method, this profile will **not persist** in the sense that the directory will remain but it won't be considered a 'profile' by Firefox (unless you run FF with this command to the same directory every time).

The directory you use as a 'profile' in this case will not be picked up by Profile Manager, whereas a profile created by default on first run of FF, or via Profile manager will be visible and available to profile manager afterwards